.circleci/config.yml

```
@@ -0,0 +1,25 @@
+version: 2.0
+jobs:
+  test:
+    docker:
+      - image: circleci/elixir:1.10.3
+        environment:
+          MIX_ENV: test
+
+    working_directory: ~/app
+
+    steps:
+      - checkout
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+      - run: mix deps.get
+      - run: mix format --check-formatted
+      - run: mix compile --force --warnings-as-errors
+      - run: mix test
+      - run: mix docs 2>&1 | (! grep -q "warning:")
```

I've found a way to both print and search for warnings:
```suggestion
      - run: mix docs 2>&1 | tee >(! grep -q "warning:")
```